src/tools/cephfs/shell/cephfs-shell

```diff
@@ -1633,7 +1633,7 @@ class CephFSShell(Cmd):
         """
         try:
             poutput('{}'.format(cephfs.getxattr(args.path,
-                                to_bytes(args.name)).decode('utf-8')))
+                                to_bytes(args.name), size=65536).decode('utf-8')))
```

Comment:
Is this a problem with the xattr name or value ?
Please mention the same as part of the commit message as the description in the tracker is confusing too.

Reply:
The problem is about length of xattr's value.when the value's length >256, report this  error.I've read code of function getxattr，default value of size is 256，so I set it to 65536 like linux.

Reply:
Linux uses MAX_SIZE as power 16, seems ok to me.

Reply:
Please rename the commit title to:
**cephfs-shell: bump up acceptable xattr value len to 64K**

Also, remove the spurious line in the commit message:
`Please enter the commit message for your changes. Lines starting`

Reply:
@mchangir  Title has been modified，thanks for the review.

Reply:
> The problem is about length of xattr's value.when the value's length >256, report this error.I've read code of function getxattr，default value of size is 256，so I set it to 65536 like linux.

I think this detail should be added to the commit message and to the tracker ticket.